root/components/EntityTabs.js

```diff
@@ -105,6 +105,10 @@ function buildLinks(
     }
   }
 
+  if (entity.entityType === 'release') {
+    links.push(buildLink(l('Edit Relationships'), entity, 'edit-relationships', page));
```

Comment:
In https://github.com/metabrainz/musicbrainz-server/blob/2baca997d9eb737aff377cc541d1ef0108cfe53f/root/release/edit_relationships.tt#L7 we pass `page => 'edit_relationships'` which I think needs to be changed to use a hyphen to match the third argument here, so that the tab is highlighted correctly once selected.

Reply:
Oh yeah, thanks :) Fixed!